### DISCUSSION TITLE
Talvez seja a hora de sair do Insomnia para o... Bruno (br-dev)

### DISCUSSION THREAD

**Discussion by giscus** [br-dev-org/site#27] [Post Discussions]
# Talvez seja a hora de sair do Insomnia para o... Bruno (br-dev)

O ciclo se repete mais uma vez, e com a burocratização do Insomnia surge uma nova opção: Bruno

https://br-dev.org/publicacoes/talvez-seja-a-hora-de-sair-do-insomnia-para-o-bruno

**Comment by raphaeldevs** [comment]
Ultimamente estou insatisfeito com o Insomnia. Várias vezes precisei fazer alguma coisa rápida e me frustrei porque tive que fechar uns popups chatos pra fazer login.

**Reply by raphaeldevs** [reply]
![image](https://github.com/br-dev-org/site/assets/64565938/bc12b0d2-3a00-4a77-8555-9ae0c09f2765)
ME LARGA SIO :joy:

**Comment by raphaeldevs** [comment]
Um dica legal: se você gosta de criar collections no Insomnia como eu, pode exportá-las e importar no Bruno, ele consegue importar collections do Insomnia.

**Reply by PabloDinella** [reply]
Bom saber!
